src/components/grapher/GrapherDots.vue

```diff
@@ -2,7 +2,7 @@
   <g class="grapher-dots--dots-container">
     <circle
       v-for="dot in stuntSheetDotsUnselected"
-      :key="`${dot.x}-${dot.y}-dots--dot`"
+      :key="`${dot.id}-dots--dot`"
```

Comment:
this is good because it allows us to have dots in the same place!

Reply:
![image](https://user-images.githubusercontent.com/3744644/103470608-37cb4b80-4d29-11eb-9816-c3a0073e6605.png)